doc/protocols.rst

```diff
@@ -0,0 +1,124 @@
+.. py:currentmodule:: lsst.ts.ess
+
+.. _lsst.ts.ess-protocols:
+
+#######################
+Communication Protocols
+#######################
+
+This page describes the communication protocols used by the `EssCSC` to communicate with the ``ts_envsensors`` code running on the Raspberry Pi 4's.
+All communication is done via JSON encoded strings.
+All JSON strings have a common header, indicated by the ``msg_type`` keyword.
+The ``msg_type`` keyword can have the values
+
+- command, indicating a command
+- response, indicating a response to a command
+- telemetry, indicating telemetry sent by the Raspberry Pi 4's
```

Comment:
Do we have the type of "event" need to consider?

Reply:
I don't think so. I intend to have the CSC generate the events based on the telemetry it receives and not have the Raspberry Pi's generate the events.